tests/hardware_reboot/testdata/hardware_usb_reboot.txt

```diff
@@ -0,0 +1,83 @@
+# Simple test of USB passthrough functionality after reboot of guest
+
+{{$usb_dev := "2-2"}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+
+eden pod deploy -n n11 --memory=512MB docker://itmoeve/eclient:0.4 -p 2223:22 --adapters USB2:2
+test eden.app.test -test.v -timewait 20m RUNNING n11
+
+exec -t 20m bash ssh.sh 2223
+stdout 'Ubuntu'
+
+exec -t 20m bash get-lshw.sh 2223
+cp stdout before_reboot
+
+exec -t 20m bash get-usb.sh 2223
+grep 'QEMU USB HARDDRIVE' {{$usb_dev}}.usb.product
+
+# reboot from guest
+exec -t 20m bash reboot.sh 2223
+exec sleep 10
+
+exec -t 20m bash ssh.sh 2223
+stdout 'Ubuntu'
```

Comment:
How do you know it rebooted? 
FWIW in the ztests we check last (last -n 1 --time-format iso reboot | awk '{print $5}'| head -1) before the reboot and then wait for that timestamp to increase.

Reply:
Actually we have a special reboot-detector eden.reboot.test (tests/reboot/reboot_test.go), which we using for ex. in tests/workflow/testdata/reboot_test.txt. In this detector we checking the lastRebootTime from dinfo. The main advantage is that we don't need SSH access.

Reply:
@sadov @eriknordmark Good point, I will apply checks.